frontend/app/components/redesign/components/footer/EnhancedFooter.tsx

```diff
@@ -95,27 +98,39 @@ export const EnhancedFooter = () => {
             Terms & Conditions
           </BodyStandardLink>
         </div>
-      </div>
+      </nav>
 
-      <div className="flex md:hidden flex-col gap-md items-center justify-start w-full">
+      <section
+        className="flex md:hidden flex-col gap-md items-center justify-start w-full"
+        aria-label="Mobile footer content"
+      >
         <div className="flex flex-row gap-sm items-center justify-start">
           <img src={wmLogo} alt="Web Monetization Logo" />
           <p className="text-style-small-standard">Web Monetization</p>
         </div>
-        <div className="w-full h-px bg-purple-100" />
+        <div className="w-full h-px bg-purple-100" aria-hidden="true" />
         <SocialLinks className="w-full flex flex-row items-center justify-between" />
-      </div>
+      </section>
 
       <SocialLinks className="hidden md:flex w-[200px] flex-row items-center justify-between" />
 
-      <div className="hidden md:block w-full h-px bg-purple-100" />
+      <div
+        className="hidden md:block w-full h-px bg-purple-100"
+        aria-hidden="true"
+      />
 
-      <div className="hidden md:flex flex-row gap-sm items-center justify-start">
+      <section
+        className="hidden md:flex flex-row gap-sm items-center justify-start"
+        aria-label="Branding"
+      >
```

Comment:
It's ok without separate section. Having too many sections is confusing for users who browse via landmarks. Use section for where it makes sense for it to be announced separately.